lib/ruby-stackoverflow/client.rb

```diff
@@ -58,7 +58,7 @@ def append_params_to_url(url, options)
     end
 
     def key_params
-      {key: configuration.client_key, site: 'gamedev', access_token: configuration.access_token}
+      {key: configuration.client_key, site: 'gamedev', pagesize: 100, access_token: configuration.access_token}
```

Comment:
Not sure if we want to fetch 100 for each request. We may need to add this param only for questions as default. This change may be the reason our suite is broken again: everything changed. Any thoughts?